src/NamedTupleUtilities.jl

```diff
@@ -0,0 +1,142 @@
+module NamedTupleUtilities
+
+import Base.startswith, Base.endswith, Base.occursin
```

Comment:
I believe we had said we would also not extend these functions, right? In that case, just delete this line here, and everything should work :)

Reply:
Just to elaborate on this one here: if you ``import`` these functions, then if you define a method of that name later in this module, it will attach that method to the imported function.

If you *don't* import these names here, then writing ``function startswith`` later here will create a new function in the module ``NamedTupleUtilities``, which is what we want.